test/html_entities_test.exs

```diff
@@ -18,6 +18,10 @@ defmodule HtmlEntitiesTest do
   end
 
   test "Encoding does replace unsafe characters" do
-    assert encode("'\"&<>") == "&apos;&quot;&amp;&lt;&gt;"
+    assert encode("'\"&<>") == "&#39;&quot;&amp;&lt;&gt;"
```

Comment:
To clarify behaviour, we should add an encoded character and assert that it _does_ double encode.
```suggestion
    assert encode("'\"&<>&amp;") == "&#39;&quot;&amp;&lt;&gt;&amp;amp;"
```